build/media-suite_helper.sh

```diff
@@ -1743,6 +1743,25 @@ EOF
             cat > "$LOCALDESTDIR"/bin/ab-pkg-config-static.bat
 }
 
+create_cmake_toolchain() {
+    local _win_path_LOCALDESTDIR="$(cygpath -m $LOCALDESTDIR)"
+    local _win_path_MINGW_PREFIX="$(cygpath -m $MINGW_PREFIX)"
+    local toolchain_file=(
+        "SET(CMAKE_RC_COMPILER_INIT windres)"
+        "SET(PKG_CONFIG_EXECUTABLE $_win_path_LOCALDESTDIR/bin/ab-pkg-config-static.bat)"
+        ""
+        "LIST(APPEND CMAKE_PROGRAM_PATH $_win_path_LOCALDESTDIR/bin ...)"
```

Comment:
What's with the triple dot?

Reply:
Probably shitty copy/paste from docs and thinking they do something like in JS.